src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -238,7 +238,7 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
 
             datasets = []
             saved_refrence_tilt_frame = None
-            
+
```

Comment:
Can you tell me which tool you're using to be able to see these formatting issues? I'm totally fine with these formatting changes, it's just that flake8 doesn't show them to me so I have no easy way of checking for them when I make or review PRs

Reply:
I think these changes are being made by Atom automatically.  I don't see them and have no control over them.